docs/MIGRATING.md

```diff
@@ -1,6 +1,238 @@
-## Migration guide
+# Migration guide
 
-This document includes high level steps describing how to migrate between high level version of the Datadog .NET Client Library (.NET Tracer)
+This document includes high level steps describing how to migrate between high level versions of the Datadog .NET Client Library (.NET Tracer)
+
+- [Migrating from v2.x.x to v3.x.x](#migrating-from-v2xx-to-v3xx)
+- [Migrating from v1.x.x to v2.x.x](#migrating-from-v1xx-to-v2xx)
+
+
+## Migrating from v2.x.x to v3.x.x
+
+The .NET tracer v3.0.0 includes breaking changes that you must be aware of before upgrading your applications. The most important high-level changes are listed below, and described in more detail later in this document
+
+- Breaking changes
+	- **Custom-only** tracing (using the _Datadog.Trace_ NuGet package), _without_ any automatic tracing, will no longer be supported. Custom instrumentation with the  _Datadog.Trace_ NuGet where you have _also_ configured [automatic-instrumentation](https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/) is still supported as it was in v2.x.x.
+	- **The public API surface has changed** in the *Datadog.Trace* NuGet package. A number of previously obsolete APIs have been removed, and some other APIs have been marked obsolete. Most changes are related to how you create `TracerSettings`  and `Tracer` instances.
+	- **.NET Core 2.0** will no longer work with automatic or custom instrumentation. This has been marked [End Of Life (EOL)](https://docs.datadoghq.com/tracing/trace_collection/compatibility/dotnet-core/#supported-net-core-runtimes)  by Microsoft since 2018, and will no longer work at all in v3.0.0+ of the Datadog .NET tracer.
+	- **Changes to default settings**. The default values of some settings have changed, and others have been removed. See below for more details.
+	- **Changes in behavior**. The semantic requirements and meaning of some settings have changed, as have some of the tags added to traces.  See below for more details.
+	- **The 32-bit MSI installer will no longer be available**. The 64-bit MSI installer already includes support for tracing 32-bit processes, so you should use this installer instead. 
+- Deprecation notices
+	- **.NET Core 2.1 is marked EOL** in v3.0.0+ of the tracer. That means versions  2.0, 2.1, 2.2 and 3.0 of .NET Core are now EOL. .NET Core 2.1 may still work with v3.0.0+, but is will no longer receive significant testing and you will receive limited support for issues arising with EOL versions.
```

Comment:
```suggestion
	- **.NET Core 2.1 is marked EOL** in v3.0.0+ of the tracer. That means versions  2.1, 2.2 and 3.0 of .NET Core are now EOL. .NET Core 2.1 may still work with v3.0.0+, but is will no longer receive significant testing and you will receive limited support for issues arising with EOL versions.
```

I wouldn't mention 2.0 here since you've stated above that it doesn't work at all. I feel like it would bring confusion.